passes/cmds/select.cc

```diff
@@ -1065,6 +1065,10 @@ struct SelectPass : public Pass {
 		log("        selection is non-empty. i.e. produce an error if no object or module\n");
 		log("        matching the selection is found.\n");
 		log("\n");
+		log("    -assert-mod-count N\n");
+		log("        do not modify the current selection. instead assert that the given\n");
+		log("        selection contains exactly N modules.\n");
```

Comment:
Should we add a note that this will count partially selected modules, and not check that the module is fully selected? Right now it's ambiguous from the description what exactly the "contains" relationship is.

Reply:
I think we should. Let's do

> do not modify the current selection. instead assert that the given selection contains exactly N modules (partially or fully selected)